### DISCUSSION TITLE
[Art competition] Padding Oracle Attack

### DISCUSSION THREAD

**Discussion by frogfrog2** [prosyslab-classroom/cs348-information-security#404] [Art competition]
![dream_TradingCard (15)](https://github.com/prosyslab-classroom/cs348-information-security/assets/126472555/e7d1d9bb-15fc-459f-815b-317d98fedc58)

## Short Description
For me, "Padding Oracle Attack" means "A man who tries to take all the bread away from an angel handing out bread".

## Long Description (up to 3 sentences)
Padding oracle is a service provided for user convenience, but attackers actively use the oracle to steal the plaintext in the padding oracle attack. In this respect, I noted that attackers attack someone by exploiting the goodwill given to them. I represented this as a man trying to take all the bread away from an angel handing out a piece of bread for each person.

**Comment by KihongHeo** [comment]
Good. It is unfortunate that this AI does not clearly understand the structure of the human body.
